widgetastic_manageiq/__init__.py

```diff
@@ -5349,3 +5349,24 @@ def fill(self, value):
         self.browser.send_keys(value, self)
         self.browser.send_keys(Keys.ENTER, self)
         return True
+
+
+class AutomateRadioGroup(RadioGroup):
+    LABELS = ".//label"
+    BUTTON = './/label[normalize-space(.)={}]/preceding-sibling::input[@type="radio"][1]'
+
+    @property
+    def selected(self):
+        names = self.button_names
+        for name in names:
+            bttn = self.browser.element(self.BUTTON.format(quote(name)))
+            if bttn.get_attribute("checked") is not None:
+                return name
+        else:
+            return names[0]
```

Comment:
I see you have been just moving this piece of code, but it doesn't seem to do what I would expect. I find strange it returns `names[0]`if it finds no item is selected.

Reply:
aaaaaaahhh thanks @jaryn . I just moved but thank need to check it. :+1: